src/Symfony/Component/Serializer/Tests/Normalizer/ObjectNormalizerTest.php

```diff
@@ -99,6 +102,15 @@ private function createNormalizer(array $defaultContext = [], ?ClassMetadataFact
         $this->normalizer->setSerializer($this->serializer);
     }
 
+    public function testNamedArgumentsOnInterfaceWillNotThrowError()
```

Comment:
this test should be removed IMO as it gives the false impression that we guarantee backwards compatibility on named arguments (which we don't)